src/routing/admin/organisation/OrganisationAuthorizationRoute.tsx

```diff
@@ -12,8 +13,11 @@ const OrganisationAuthorizationRoute: FC<OrganisationAuthorizationRouteProps> =
 
   return (
     <Switch>
-      <Route exact path={`${path}/:role`}>
-        <OrganisationAuthorizationPage paths={currentPaths} />
+      <Route exact path={`${path}/admins/:role`}>
+        <OrganisationAdminAuthorizationPage paths={currentPaths} />
+      </Route>
+      <Route exact path={`${path}/owners/:role`}>
+        <OrganisationOwnerAuthorizationPage paths={currentPaths} />
```

Comment:
the `:role` parameter is redundant. The previous part of the path contains the actual role and the parameter is never used.